plugin.video.roosterteeth/resources/lib/roosterteeth_const.py

```diff
@@ -79,4 +78,14 @@ def log(name_object, object):
 
 def getSoup(html, default_parser="html5lib"):
     soup = BeautifulSoup(html, default_parser)
-    return soup
\ No newline at end of file
+    return soup
+
+
+def find_all(string_to_be_searched_for, string_to_be_searched_in):
+    start = 0
+    while True:
```

Comment:
maybe `while not xbmc.Monitor().abortRequested()`?

Reply:
i adjusted it a bit. Making a new pr.